ReactiveObjC/RACSubscriber.h

```diff
@@ -46,6 +46,6 @@
 /// A subscriber may receive multiple disposables if it gets subscribed to
 /// multiple signals; however, any error or completed events must terminate _all_
 /// subscriptions.
-- (void)didSubscribeWithDisposable:(RACCompoundDisposable *)disposable;
+- (void)didSubscribeWithDisposable:(RACCompoundDisposable * _Nonnull)disposable;
```

Comment:
Could we use `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END` around the protocol instead of explicitly specifying `_Nonnull` here?